documentation/assemblies/security/assembly-using-your-own-certificates.adoc

```diff
@@ -0,0 +1,31 @@
+// This assembly is included in the following assemblies:
+//
+// security/assembly-security.adoc
+
+[id='security-using-your-own-certificates-{context}']
+= Using your own CA certificates and private keys
+
+[role="_abstract"]
+Install and use your own CA certificates and private keys instead of using the defaults generated by the Cluster Operator.
+You can replace the cluster and clients CA certificates and private keys.
+
+You can switch to using your own CA certificates and private keys in the following ways:
+
+* Install your own CA certificates and private keys before deploying your Kafka cluster
+* Replace the default CA certificates and private keys with your own after deploying a Kafka cluster
```

Comment:
Should we make it clear that the replacing happens essentially through the `Renew CA certificates and private keys`process?

Reply:
We had something on line 25. I've moved that up and added something to the bullet too